dataeng/jobs/analytics/SnowflakeReplicaImportFromS3.groovy

```diff
@@ -69,23 +69,16 @@ class SnowflakeReplicaImportFromS3 {
                 wrappers common_wrappers(allVars)
                 steps {
                     shell(dslFactory.readFileFromWorkspace('dataeng/resources/snowflake-replica-import.sh'))
-                    conditionalSteps {
-                        condition { not { fileExists('build_skipped', BaseDir.WORKSPACE) } }
-                        // If the condition fails (i.e. not-not fileExists is equivalent to fileExists), set the build
-                        // status to unstable.
-                        runner('Unstable')
-                    }
```

Comment:
I'm giving up on the whole "unstable" status because I just can't get it to work, experimentally.